### PR TITLE
{ServiceBus} `az servicebus queue` `az serviebus topic` `az servicebus topic subscripiton`: Migrating servicebus queue, servicebus topic, servicebus topic subscription cmdlets to AAZ

### PR DESCRIPTION
**Pull request by schaudhari6254888**
**Related command**

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
service bus

File: src/azure-cli/azure/cli/command_modules/servicebus/custom.py [hunk 1]
```diff
@@ -631,10 +324,6 @@ def cli_migration_complete(client, resource_group_name, namespace_name, config_n
     return client.complete_migration(resource_group_name, namespace_name, config_name)
 
 
-def revert(client, resource_group_name, namespace_name, config_name="$default"):
-    return client.revert(resource_group_name, namespace_name, config_name)
-
-
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/servicebus/custom.py:637]: The `revert` function is removed here

**Comment by schaudhari6254888** [comment]
This PR is for Migrating ServiceBus queue, topic, rule cmdlets to aaz.
Can someone please look into is & review this.
Thank you!

**Comment by evelyn-ys** [comment]
Hi @schaudhari6254888, I just merged #24835 and it brings many conflicts to this PR. Can you resolve first?

**Comment by schaudhari6254888** [comment]
> Hi @schaudhari6254888, I just merged #24835 and it brings many conflicts to this PR. Can you resolve first?

Hi @evelyn-ys , Resolved.

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**